tests/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks.csproj

```diff
@@ -30,7 +30,7 @@
   </Choose>
   
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" />
-    <PackageReference Include="SkiaSharp" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.12" />
```

Comment:
You don't want the version number here. That's handled in the targets file.

Reply:
I believe it was added by Nuget, will fix it

Reply:
Thanks!